src/editor.rs

```diff
@@ -422,6 +424,10 @@ impl Editor {
     /// Try to load a file. If found, load the rows and update the render and syntax highlighting.
     /// If not found, do not return an error.
     fn load(&mut self, path: &Path) -> Result<(), Error> {
+        if !std::fs::metadata(path)?.file_type().is_file() {
```

Comment:
I think we should also keep supporting symlinks  ([is_symlink](https://doc.rust-lang.org/std/fs/struct.Metadata.html#method.is_symlink)) as kibi seems to be working well with them

Reply:
good point!